src/AutoRest.CSharp/Common/Generation/Writers/CodeWriter.cs

```diff
@@ -661,10 +661,20 @@ public string ToString(bool header)
             }
 
             // Normalize newlines
-            builder.AppendLine(new string(_builder.AsSpan(0, _length).Trim()).Replace(_newLine, Environment.NewLine));
```

Comment:
Did we lose the normalization here or does AppendLine on line 669 always use Environment.NewLine?

Reply:
yes - it uses Env.NewLine